src/applications/vaos/services/appointment/transformers.js

```diff
@@ -179,13 +193,32 @@ function getAppointmentDuration(appt) {
  * @param {Object} appt  VAR appointment object
  * @returns {Array} Array of participants of FHIR appointment
  */
-function addParticipantAndContained(appt, transformed) {
-  const isVideo = isVideoVisit(appt);
-  let participant;
-  let contained;
+function setParticipant(appt) {
+  if (isCommunityCare(appt)) {
+    const address = appt.address;
+    return [
+      {
+        actor: {
+          name: appt.providerPractice,
```

Comment:
There's also a provider first and last name in the CC data, which we show in the cancel modal. I don't have any idea how they'll map that, but I would just shove it in this resource somewhere for now.

Reply:
Added to `contained` with temp practictioner id.